code/modules/organs/pain.dm

```diff
@@ -47,7 +47,7 @@
 	if(.)
 		var/force_emote = species.get_pain_emote(src, power)
 		if(force_emote && prob(power))
-			var/decl/emote/use_emote = usable_emotes[force_emote]
+			var/decl/emote/use_emote = GET_DECL(force_emote)
```

Comment:
`get_pain_emote` still returns the key rather than the decl typepath. This will runtime, I think.